manim/mobject/text/numbers.py

```diff
@@ -214,17 +226,21 @@ def _get_num_string(self, number):
 
         return num_string
 
-    def _string_to_mob(self, string: str, mob_class: VMobject | None = None, **kwargs):
+    def _string_to_mob(
+        self, string: str, mob_class: type[TextLike] | None = None, **kwargs: Any
+    ) -> TextLike:
```

Comment:
I wouldn't worry about trying to get this behavior type hinted - it's far too dynamic for typecheckers to understand.

Reply:
I see, thanks for the feedback. I'll leave it as is, then.